src/core/Group.cpp

```diff
@@ -440,6 +459,18 @@ QList<Entry*> Group::entriesRecursive(bool includeHistoryItems) const
     return entryList;
 }
 
+Entry* Group::findEntry(const Uuid& uuid)
+{
+    Q_ASSERT(!uuid.isNull());
+    Q_FOREACH (Entry* entry, m_entries) {
```

Comment:
We need to convert Q_FOREACH to C++11 foreach. (Q_FOREACH will be deprecated soon)